howto.md

```diff
@@ -1,14 +1,31 @@
 ---
-title: How to use them
+title: How to Run a Delta Chat Bot
 ---
 
-# How to use
+# How to Run a Delta Chat Bot
+
+To run a bot, you don't need to be a programmer. 
+You don't even need to own or rent a server.
```

Comment:
Also you don't need any **port-forwarding** or changing of other router settings.

Reply:
the point is that read text from top to bottom, which means the first text needs to interesting that you keep reading, so I would move the key points up